frontend/src/component/common/TooltipLink/TooltipLink.tsx

```diff
@@ -6,32 +6,49 @@ import {
 } from '../HtmlTooltip/HtmlTooltip';
 
 const StyledLink = styled(Link, {
-    shouldForwardProp: (prop) => prop !== 'highlighted',
-})<{ highlighted?: boolean }>(({ theme, highlighted }) => ({
-    backgroundColor: highlighted ? theme.palette.highlight : 'transparent',
-    color: theme.palette.text.primary,
-    textDecorationColor: theme.palette.text.disabled,
-    textDecorationStyle: 'dashed',
-    textUnderlineOffset: theme.spacing(0.5),
-    whiteSpace: 'nowrap',
-}));
+    shouldForwardProp: (prop) => prop !== 'highlighted' && prop !== 'clampText',
+})<{ highlighted?: boolean; clampText?: boolean }>(
+    ({ theme, highlighted, clampText }) => ({
+        backgroundColor: highlighted ? theme.palette.highlight : 'transparent',
+        color: theme.palette.text.primary,
+        textDecorationColor: theme.palette.text.disabled,
+        textDecorationStyle: 'dashed',
+        textUnderlineOffset: theme.spacing(0.5),
+        whiteSpace: 'nowrap',
+        ...(clampText
+            ? {
+                  overflow: 'hidden',
+                  textOverflow: 'ellipsis',
+                  maxWidth: '100%',
+                  display: 'block',
+              }
+            : {}),
+    }),
+);
 
 interface ITooltipLinkProps extends LinkProps {
     tooltip: ReactNode;
     highlighted?: boolean;
     tooltipProps?: Omit<IHtmlTooltipProps, 'title' | 'children'>;
     children: ReactNode;
+    clampText?: boolean;
 }
 
 export const TooltipLink = ({
     tooltip,
     highlighted,
     tooltipProps,
     children,
+    clampText,
```

Comment:
does it need to be configurable? when do we want ellipsis vs scrollbar?

Reply:
Same question 🙋🏼 But also: is it okay that we don't show the whole text. should we reconfigure the tables instead? 🤔